lib/pages/search/view/search_page.dart

```diff
@@ -0,0 +1,101 @@
+import 'package:acs_upb_mobile/pages/classes/model/class.dart';
+import 'package:acs_upb_mobile/pages/people/model/person.dart';
+import 'package:acs_upb_mobile/pages/people/service/person_provider.dart';
+import 'package:acs_upb_mobile/widgets/scaffold.dart';
+import 'package:acs_upb_mobile/widgets/search_bar.dart';
+import 'package:dynamic_text_highlighting/dynamic_text_highlighting.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:acs_upb_mobile/generated/l10n.dart';
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+
+class SearchPage extends StatefulWidget{
+  @override
+  State<StatefulWidget> createState() => _SearchPageState();
+}
+
+class _SearchPageState extends State<SearchPage>{
+  String filter = '';
+  List<Class> classesSearched;
+  bool searchClosed = true;
+  Future<List<Person>> people;
+  List<Person> peopleData;
+
+  @override
+  void initState() {
+    super.initState();
+    final personProvider = Provider.of<PersonProvider>(context, listen: false);
+    people = personProvider.fetchPeople();
```

Comment:
You shouldn't need to fetch people here. You should just send the query to the provider when the user submits the query (presses the search button to submit etc.). Otherwise this will get very complicated and inefficient very fast, imagine needing to fetch EVERYTHING (people, events, classes etc. just because the user opened the search page).